### ISSUE TITLE
A silly character preferences bug

### ISSUE THREAD

**Issue by Midaychi** [ParadiseSS13/Paradise#4764]
**Problem Description**:
Because of how dialogue windows work, you can que up dialogue windows in advance. This means that you can do things like be a vox with a computer monitor head or a grey with a pompadour.

**What did you expect to happen**: 
Not being able to do this.

**What happened instead**:
![Alt text](https://dl.dropbox.com/s/n9fstvgc7k5wty4/ayylmao.jpg"Ayyyy LMaaaaaao")

**Why is this bad/What are the consequences:**
It's not really, but if you're really going to have species restrictions on bits and kebabs then you should probably enforce them.

**Steps to reproduce the problem**:
1. Pick a species with bits and kebabs (IE machine person) and save it as a preference, then load a new preference slot and set it to another species and save it
2. Load the original slot and click on the part you want to (a)buse to bring up dialogue window
3. Load the other slot, and then select options in the dialogue window. Ta-Dah.

You can also do this using JUST dialogue windows, just click on the species button and then while that's up, click on the parts button and then switch the species. Part's selection of the previous species should come up.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Anything else you can tell us.

**Comment by Spacemanspark** [comment]
That's rather mortifying.

**Comment by Crazylemon64** [comment]
'ia! 'ia! Binary Yg'Oggoth's Nightmare Domain strikes again!

**Comment by tristan1333** [comment]
I hope no one fixes this

**Comment by Midaychi** [comment]
I don't have slime person unlocked, but I assume you can be a cybernetic slime or use slime bits on other's using this.

**Comment by Plotron** [comment]
I tried to have Vox hair on a human, spawned bald.

**Comment by Midaychi** [comment]
Yeah there are some post-spawn sanity checks but most of them are accidental carry-overs from copy-pasting vg code.

**Comment by Plotron** [comment]
I did manage to have Vox hair as a human ghost/observer once, though.

**Comment by Sirryan2002** [comment]
still an issue

**Comment by Spaghetti-bit** [comment]
Still an issue.

**Comment by Spaghetti-bit** [comment]
This would require a refactor of our character creator.

**Comment by poobsie** [comment]
![Image](https://github.com/user-attachments/assets/6d067a83-c7b7-4bb7-bfd1-dc568e69ddfa)

going to copy the image from the OP here since it's a dropbox link and could conceivably be lost

@Burzah, requesting this be renamed to something like "Character creator - species restrictions can be bypassed with queued dialog windows"
